### PR TITLE
feat: AWS Lambda deployment script and fixes

### PR DESCRIPTION
**Pull request by EmperorOrokuSaki**

### CODE REVIEW

File: serverless/src/functions/builds/handler.ts [hunk 1]
```diff
@@ -27,7 +27,8 @@ export const submitBuildInfo = async (
       ipfsHash: data.ipfsHash,
       domain: data.domain,
     };
-
+    console.log(buildInfo);
```
Comment (zoruka) [inline serverless/src/functions/builds/handler.ts:30]: maybe leftover

File: serverless/src/functions/builds/handler.ts [hunk 2]
```diff
@@ -37,8 +38,12 @@ export const submitBuildInfo = async (
         domain: buildInfo.domain,
       },
     });
+    console.log(buildRecord);
```
Comment (zoruka) [inline serverless/src/functions/builds/handler.ts:41]: maybe leftover

File: serverless/src/functions/builds/handler.ts [hunk 3]
```diff
@@ -37,8 +38,12 @@ export const submitBuildInfo = async (
...
+    
 
     if (buildRecord.length == 0) {
+      console.log('here i am');
```
Comment (zoruka) [inline serverless/src/functions/builds/handler.ts:45]: maybe leftover

**Review by zoruka — APPROVED** [review]
It is good review if all the commented envs should stay like that. Also I would request @jsonsivar to eyeball this one.

File: serverless/src/functions/builds/handler.ts (hunk 1 above)
Comment (EmperorOrokuSaki) [reply serverless/src/functions/builds/handler.ts:30]: I'm not quite sure if we want to keep this or not. But I guess we can add it whenever needed, later.

File: serverless/src/functions/builds/handler.ts (hunk 3 above)
Comment (EmperorOrokuSaki) [reply serverless/src/functions/builds/handler.ts:45]: These should be overwritten by the #272

File: serverless/src/functions/builds/handler.ts (hunk 2 above)
Comment (EmperorOrokuSaki) [reply serverless/src/functions/builds/handler.ts:41]: These should be overwritten by #272

**Pull request merged**